### DISCUSSION TITLE
6.0.0-beta.6

### DISCUSSION THREAD

**Discussion by arturcic** [GitTools/GitVersion#3946] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/GitTools/GitVersion/releases/tag/6.0.0-beta.6'>6.0.0-beta.6</a>.</em>

**Comment by DSpirit** [comment]
Thanks for the update! Unfortunately my pipelines start receiving this issue:
https://github.com/GitTools/GitVersion/issues/3947

Would be glad if you could have a look.
